fe1-web/src/features/social/screens/SocialSearch.tsx

```diff
@@ -34,56 +36,52 @@ const styles = StyleSheet.create({
   } as ViewStyle,
 });
 
-const SocialSearch = (props: IPropTypes) => {
-  const { currentUserPublicKey } = props;
-  const currentLao = useSelector(selectCurrentLao);
+const SocialSearch = () => {
+  const { currentUserPublicKey } = useContext(SocialMediaContext);
+  const currentLao = SocialHooks.useCurrentLao();
 
   if (!currentLao) {
     throw new Error('Impossible to open Social media Search if you are not connected to a LAO');
   }
 
   const rollCallId = currentLao.last_tokenized_roll_call_id;
-  const attendeesSelect = makeRollCallAttendeesListSelector(rollCallId);
-  const attendees = useSelector(attendeesSelect);
+  if (!rollCallId) {
+    throw new Error(
+      'Impossible to open social media search: last tokenized roll call id is undefined',
+    );
+  }
```

Comment:
same as on the other file. be careful here, we're in an expected situation (and React will be very unhappy about the number of hooks changing)

Reply:
It no longer needs to be there since `useRollCallAttendeesById` accepts an undefined roll call id now.